client/src/ingredients/attributes.js

```diff
@@ -18,20 +20,32 @@ function RenderValues({ values }) {
 function RenderAttributes({ attributes }) {
   if (attributes.length === 0) {
     return (
-      <p>This element only supports the <a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes">
-        global attributes</a>.</p>
-    )
+      <p>
```

Comment:
Is this a prettier or eslint thing? If yes, please ignore. If no, this is very unnatural to read and I'd put it back the way it was before